src/pages/RFOX/components/Claim/ClaimStatus.tsx

```diff
@@ -51,9 +58,36 @@ export const ClaimStatus: React.FC<Pick<ClaimRouteProps, 'headerComponent'> & Cl
     onTxStatusConfirmed: handleTxConfirmed,
   })
 
+  const { data: maybeSafeTx } = useSafeTxQuery({
+    maybeSafeTxHash: txId ?? undefined,
+    accountId: confirmedQuote.stakingAssetAccountId,
+  })
+
   const bodyContent: BodyContent | null = useMemo(() => {
     if (!claimAsset) return null
 
+    if (maybeSafeTx?.isQueuedSafeTx) {
+      return {
+        key: TxStatus.Pending,
+        title: [
+          'common.safeProposalQueued',
+          {
+            currentConfirmations: maybeSafeTx?.transaction?.confirmations?.length,
+            confirmationsRequired: maybeSafeTx?.transaction?.confirmationsRequired,
+          },
+        ],
+        body: [
+          'RFOX.claimPending',
+          { amount: bnOrZero(claimAmountCryptoPrecision).toFixed(8), symbol: claimAsset.symbol },
```

Comment:
![image](https://github.com/user-attachments/assets/a4799b38-e0a8-41b6-b5a6-bb0d17656299)

All those amounts from the status page are displaying unneeded zeros (0.06250000 FOX instead of 0.0625 FOX), you could avoid this by using `toCrypto` with omitDecimalTrailingZeros